docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md

```diff
@@ -177,3 +177,46 @@ Sample response:
 }
 ```
 
+Predict request used in Neural Search:
+```json
+POST /_plugins/_ml/_predict/text_embedding/<ML Model Id>
+{
+    "text_docs":[ "today is sunny"],
+    "return_number": true,
+    "target_response": ["sentence_embedding"]
+}
+```
+
+Sample Response:
+```json
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "sentence_embedding",
+          "data_type": "FLOAT32",
+          "shape": [
+            1024
+          ],
+          "data": [
+            -0.007699765,
```

Comment:
Is this the real output from titan embedding model? 
Why it's same with multimodal-model?  

Please do a real testing , verify if this old api works and add the real output.
```
            -0.007699765,
            0.0045603644,
            -0.0044756276,
            -0.002815825,
            0.03696563,
```

Reply:
Good catch. I was thinking only the format matters. Real data are updated in the new commit.